pkg/scheduler/api/node_info/node_info.go

```diff
@@ -637,8 +637,9 @@ func getNodeGpuMemory(node *v1.Node) (int64, bool) {
 		gpuMemoryLabelValue = convertBytesToMib(gpuMemoryLabelValue)
 	}
 
-	gpuMemoryInMb := convertMibToMb(gpuMemoryLabelValue)
```

Comment:
`convertMibToMb` is not used anymore, delete it as well (probably linter will fail anyway)

Reply:
I just create a commit base on your review